app/controllers/turbo/teachers/quizzes_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Turbo
+  module Teachers
+    class QuizzesController < ApplicationController
+      def create
+        @quiz = Quiz.new(quiz_params)
+  
+        if create_quiz.success?
+          flash[:notice] = "Quiz created successfully"
+  
+          redirect_to dashboard_path
```

Comment:
HEre we are handling HTML and turbo in the same controller that should be used just fot turbo

Reply:
since the action it self does use turbo in response (not always), for now we will leave as it is